joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller_impl.h

```diff
@@ -555,7 +561,7 @@ updateTrajectoryCommand(const JointTrajectoryConstPtr& msg, RealtimeGoalHandlePt
     }
     else
     {
-      // All trajectory points are in the past, nothing new to execute. Keep on executing current trajectory
+      ROS_WARN_NAMED(name_, "Failed to initialize trajectory from message, skipping.");
```

Comment:
After actually reviewing `initJointTrajectory`, this warning seems to be superfluos.

Reply:
Agreed.